### DISCUSSION TITLE
When installing pre release it installs v1.4.31

### DISCUSSION THREAD

**Discussion by RPierrick** [sqlalchemy/sqlalchemy#7619] [Usage Questions] [answered]
Hello, 
I'm trying to install the development version of SQlAlchemy (2.0.0.b1) using the following command : 
`pip install --pre SQlAlchemy`  as indicated on the website.

But it installs the lastest version (1.4.31). 

Does someone know how to install the development version ?

Thanks

**Comment by CaselIT** [comment, marked as answer]
Hi
That would be because there is no 2.0.0.b1 published on pypi at this time.
You need to install using the git url

**Reply by RPierrick** [reply]
Thank you for your answer, now I can use the development version

**Reply by CaselIT** [reply]
please note that the current main is in development so there will be bugs and unfinished features
